app/views/settings_dialog.py

```diff
@@ -497,6 +497,12 @@ def _do_sorting_tab(self) -> None:
         explanatory_label.setWordWrap(True)
         tab_layout.addWidget(explanatory_label)
 
+        # Checkboxes
+        self.db_use_dependency_for_sorting_checkbox = QCheckBox(
+            "Use mod dependency rules for sorting such that dependencies are loaded before the dependent mod"
```

Comment:
"Use dependency rules for sorting."

Then add a tooltip for more context: "Loads defined dependencies before the dependent mod."

Reply:
Fixed